src/javascript/app_2/App/Components/Form/TimePicker/dialog.jsx

```diff
@@ -0,0 +1,71 @@
+import classNames   from 'classnames';
+import React        from 'react';
+import { localize } from '_common/localize';
+import { toMoment } from 'Utils/Date';
+
+function selectOption(type, value, props, is_enabled = true) {
+    if (is_enabled && props.value) {
+        const [ prev_hour, prev_minute ] = props.value.split(':');
+        if ((type === 'h' && value !== prev_hour) || (type === 'm' && value !== prev_minute)) {
+            props.onChange(`${type === 'h' ? value : prev_hour}:${type === 'm' ? value : prev_minute}`);
+        }
+    }
+}
+
+function Dialog(props) {
```

Comment:
Why not use the same style as our other functional stateless components? I Think it's easier for consistency if we write them the same way 😃 
```
const Dialog = ({
 destructure props here
}) => { ... }
```